### DISCUSSION TITLE
Lost 2Factor Auth key. Hence unlinked email and registered as new user with same Email. How can I go back to old userId?

### DISCUSSION THREAD

**Discussion by satheeshmkm83** [community/community#146319] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

I was holding a username satheeshmkm, I lost 2Factor authentication key for the same whn I formatted my system. As none of the reset worked, I unlinked my email from the acccount. Then created a new user with same email satheeshmkm83. Please hel;p me change back to old username satheeshmkm as most of my works are in that account

**Comment by queenofcorgis** [comment, marked as answer]
Hi there, 

Thank you for your patience. The community is unable to assist with nor escalate with this account-related situation. Please review our doc: [Recovering your account if you lose your 2FA credentials](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials). Within the doc, you also have the option to go through recovery with a Virtual Agent. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
